pom.xml

```diff
@@ -1601,8 +1601,7 @@
                         </environmentVariables>
                         <systemProperties>
                             <log4j.ignoreTCL>true</log4j.ignoreTCL>
-                            <log4j.configuration>file:src/test/resources/log4j.properties</log4j.configuration>
-                            <log4j2.configurationFile>file:src/test/resources/log4j2-test.properties</log4j2.configurationFile>
+                            <log4j2.configurationFile>file:src/test/resources/log4j2-test.xml</log4j2.configurationFile>
```

Comment:
it seems shall be `<log4j2.configurationFile>src/test/resources/log4j2-test.xml</log4j2.configurationFile>`


```scala
WARN Invalid URI file:src/test/resources/log4j2-test.xml
```

Reply:
cc @cfmcgrady @pan3793

Reply:
cc @1456173400  would you please take a look?

Reply:
fixed in #2971